tools/sil-opt/SILOpt.cpp

```diff
@@ -314,7 +314,7 @@ int main(int argc, char **argv) {
   // Setup the SIL Options.
   SILOptions &SILOpts = Invocation.getSILOptions();
   SILOpts.InlineThreshold = SILInlineThreshold;
-  SILOpts.VerifyAll = EnableSILVerifyAll;
+  SILOpts.VerifyAll = true;
```

Comment:
This (and the other changes to `LLVMOpt`/`CompilerInvocation` seem to be required to trigger issues, otherwise we don't run the verifier after each pass). Of course this can't go in as is, but it seems to be valuable to keep this hack around in the PR for testing purposes.